Source/Charts/Charts/ChartViewBase.swift

```diff
@@ -321,6 +321,14 @@ open class ChartViewBase: NSUIView, ChartDataProvider, AnimatorDelegate
 
     // MARK: - Highlighting
 
+    /// Set this to false if you don't want your highlight is deleted when users double tap on it
+    /// **default**: true
```

Comment:
/// Set this to false if you don't want your highlight beings deleted when users double tap on it